pubspec.yaml

```diff
@@ -17,6 +17,7 @@ dependencies:
   cli_util: '>=0.0.1 <0.1.0'
   glob: '>=1.0.0 <2.0.0'
   http_server: '>=0.9.0 <0.10.0'
+  path: '^1.3.5'
```

Comment:
Can we make this version constrain broader? `path` is heavily used, and I want to make sure we keep our constraints wide so that we don't contribute to unresolvable dependencies.

Perhaps `^1.0`?